gen_sample.go

```diff
@@ -122,6 +122,7 @@ func newPostgresqlStandaloneSample() *v1alpha1.PostgresqlStandalone {
 		},
 		ObjectMeta: metav1.ObjectMeta{Name: "my-instance", Namespace: "default", Generation: 1},
 		Spec: v1alpha1.PostgresqlStandaloneSpec{
+			//ConnectableInstance: v1alpha1.ConnectableInstance{WriteConnectionSecretToRef: v1alpha1.ConnectionSecretRef{Name: "my-secret"}},
```

Comment:
Is this commented out intentionally?

Reply:
ah, yes, I wanted that this field is set by the defaulter webhook when installing the sample, so I'll remove the line.